Dockerfile

```diff
@@ -0,0 +1,25 @@
+# ---- BUILD ENVIRONMENT ----- #
+
+FROM digitallivinginternational/python-base:ubuntu22.04-python3.10-nginx-node as build
```

Comment:
These should be based on images like `node`, `ubuntu`, `nginx`, `python` etc., not our base images.